CONTRIBUTING.md

```diff
@@ -265,6 +265,22 @@ severely complicates Git history. It is intended for special circumstances, as w
 the PR branch cannot or should not be modified. If you have any questions about it,
 please ask us.
 
+### Pull Requests & Issues
+
+A pull request *fixes* a problem that is *described* in an issue. Make sure
+to file an issue before opening a pull request, bar fom trivial PRs. In the
```

Comment:
```suggestion
to file an issue before opening a pull request, aside from trivial PRs. In the
```
I'd also suggest that such "trivial" PRs explain their rationale, since it's a judgement call.